org-code-javabuilder/lib/src/test/java/org/code/javabuilder/CodeBuilderTest.java

```diff
@@ -19,32 +16,18 @@
   private UserProjectFiles userProjectFiles;
   private GlobalProtocol globalProtocol;
   private JavabuilderFileManager fileManager;
+  private File tempFolder;
   private CodeBuilder codeBuilder;
 
   @BeforeEach
   public void setUp() throws Exception {
     globalProtocol = mock(GlobalProtocol.class);
     fileManager = mock(JavabuilderFileManager.class);
+    tempFolder = mock(File.class);
     when(globalProtocol.getFileManager()).thenReturn(fileManager);
     userProjectFiles = mock(UserProjectFiles.class);
 
-    codeBuilder = new CodeBuilder(globalProtocol, userProjectFiles);
-  }
-
-  @Test
-  void replacesSystemIOAndClearsTempWhenCloseCalled() {
```

Comment:
Should something similar to this test be moved to `CodeExecutionManager`?

Reply:
Hmm good call, I'll add a test.